commit c3b3b538666cfa2a33f231e8ffb7424761f1a638
Author: Matus Kral <matuskral@me.com>
Date:   Tue Dec 29 15:43:03 2015 +0100

    fix for HB1

the SR's uboot tree after commit 144b1e90d15f3a94f0c8183827d4f1b93f6f2e68
is keeping IPU started - what is keeping gfx in 4.x+ kernels from starting 

diff --git a/arch/arm/imx-common/cpu.c b/arch/arm/imx-common/cpu.c
index 5c3c080..ca74901 100644
--- a/arch/arm/imx-common/cpu.c
+++ b/arch/arm/imx-common/cpu.c
@@ -176,5 +176,9 @@ void arch_preboot_os(void)
 {
 	/* disable video before launching O/S */
 	ipuv3_fb_shutdown();
+
+	struct mxc_ccm_reg *ccm = (struct mxc_ccm_reg *)CCM_BASE_ADDR;
+	setbits_le32(&ccm->analog_pll_video, BM_ANADIG_PLL_VIDEO_POWERDOWN);
+
 }
 #endif

